maggy/maggy/services/executor.py

```
@@ -0,0 +1,253 @@
+                proc = await asyncio.create_subprocess_exec(
+                    "python3", "-m", "scripts.icpg.symbols", "--keyword", kw, "--limit", "5",
+                    stdout=asyncio.subprocess.PIPE,
+                    stderr=asyncio.subprocess.PIPE,
+                    cwd=str(bootstrap_path),
+                )
+                stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=5)
+                text = (stdout or b"").decode("utf-8", errors="replace").strip()
+                if text:
+                    symbols_out.append(f"- keyword `{kw}`: {text[:300]}")
+            except Exception:
+                continue
```

Ruff (0.15.10)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)